src/datomic_rest_api/rest/gene.clj

```diff
@@ -551,22 +560,126 @@
    :description
    "phenotype based on interaction"}))
         
-   
+(defn drives-overexpression [gene]
+  (let [db (d/entity-db gene)]
```

Comment:
I will fix this for the next release. Still working on this section of the code.